src/components/CMessageRow.vue

```diff
@@ -52,7 +52,7 @@
 </template>
 
 <script>
-import { format, formatDistance } from 'date-fns';
+import { format, formatDistance, formatDistanceStrict } from 'date-fns';
```

Comment:
I think it's not precise enougth; because 1h59min would appear as 1h. Write your own function:
should return hh:mm:ss.